src/xdocs/checks/naming/parametername.xml

```diff
@@ -179,13 +179,11 @@ class MyClass {
         </source>
         <p id="Example5-code">Code Example:</p>
         <source>
-class MyClass {
-  void method1(int v1) {} // OK
-  protected method2(int V2) {} // violation, Parameter name 'V2'
-                               // must match pattern '^[a-z]([a-z0-9][a-zA-Z0-9]*)?$'
-  private method3(int a) {} // OK
-  public method4(int b) {} // violation, Parameter name 'b'
-                           // must match pattern '^[a-z][a-z0-9][a-zA-Z0-9]*$'
+class Example5 {
+  void method1(int v1) {}
+  private void method2(int V) {} // violation &quot;Parameter name 'V' must match pattern&quot;
```

Comment:
Why did we lose protected?
Also why did we lose the 2 in the parameter name?

Reply:
just to maintain the required length of line as fixed in the config. of checkstyle. that was `85` or something. But the lenght was exceeding the maximum extent and was `87` or something. `private` and `protected` following the same pattern thats why I thought to make it private. Also in the `ParameterNameCheckTestExample.java` file line length was exceeding maxium extent of `100` words thats why I changed `V2` to `V`

Reply:
The other examples in this PR removed the violation messages except for this one file.

@romani Are we keeping the violation messages or not?

Reply:
We keep only when it matters.
In this check only when we override message.
In other cases it is too obvious.

Reply:
>Why did we lose protected?
Also why did we lose the 2 in the parameter name?

This is not expected. We need to restore.
We can always shorten something else in method declaration or...

Please share a problem and we will suggest how to handle it.

Reply:
Can we change the methods name here to maintain the line size issue?

Reply:
```
class Example5 {
  void fun1(int v1) {}
  protected void fun2(int V2) {} // violation "Parameter name 'V' must match pattern"
  private void fun3(int a) {}
  public void fun4(int b) {} // violation "Parameter name 'b' must match pattern"
```

Like this?

Reply:
good for me, please put this in code to see in diff more clearly that nothing is lost

Reply:
Okay

Reply:
Done with the changes. Review it once please